### ISSUE TITLE
[fltk] build failure

### ISSUE THREAD

**Issue by efrainceh** [microsoft/vcpkg#17561]
**Host Environment**
- OS: MacOS Big Sur 11.2.3
- Compiler: clang

**To Reproduce**
`./vcpkg install fltk`

**Failure logs** 
-Cut and past the appropriate build messages from the console output.

-- Building x64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:146 (message):
    Command failed: /Applications/vcpkg/downloads/tools/cmake-3.20.1-osx/cmake-3.20.1-macos-universal/CMake.app/Contents/bin/cmake --build . --config Debug --target install -- -v -j5
    Working Directory: /Applications/vcpkg/buildtrees/fltk/x64-osx-dbg
    See logs for more information:
      /Applications/vcpkg/buildtrees/fltk/install-x64-osx-dbg-out.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_build_cmake.cmake:105 (vcpkg_execute_build_process)
  scripts/cmake/vcpkg_install_cmake.cmake:45 (vcpkg_build_cmake)
  ports/fltk/portfile.cmake:42 (vcpkg_install_cmake)
  scripts/ports.cmake:142 (include)

Error: Building package fltk:x64-osx failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `./vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: fltk:x64-osx
  Vcpkg version: 2021-01-13-unknownhash

-Please attach any additional failure logs mentioned in the console output.

[config-x64-osx-dbg-err.log](https://github.com/microsoft/vcpkg/files/6394455/config-x64-osx-dbg-err.log)
[config-x64-osx-dbg-out.log](https://github.com/microsoft/vcpkg/files/6394456/config-x64-osx-dbg-out.log)

**Additional context**
- vcpkg is up to date (was recently installed)
- Reinstalled cmake. 
   Previous: version 3.7.1
   Current: version 3.20.1
- New to coding, so not quite sure on what I'm doing.

**Comment by JackBoosY** [comment]
Please provide failure log _/Applications/vcpkg/buildtrees/fltk/install-x64-osx-dbg-out.log_.

Thanks.

**Comment by efrainceh** [comment]
[install-x64-osx-dbg-out.log](https://github.com/microsoft/vcpkg/files/6398403/install-x64-osx-dbg-out.log)

**Comment by MoAlyousef** [comment]
https://github.com/fltk/fltk/issues/191

It seems it’s this issue. Clang on non-case sensitive filesystem mistakes FLTK’s VERSION file with the <version> standard header. This has since been fixed with FLTK 1.4 (not released yet). A solution is to enable case-sensitive filesystem on macos or to patch the FLTK sources and delete or rename the VERSION file.

**Comment by JackBoosY** [comment]
```
In file included from /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.3.sdk/usr/include/c++/v1/cstddef:37:
/Applications/vcpkg/buildtrees/fltk/src/fltk-1-7591e64af8.clean/version:1:1: error: expected unqualified-id
1.3.5
^
```

I think we should try to update this port.
